style.css

```diff
@@ -1,3 +1,51 @@
+#nav-menu {
+  display: none;
+}
+
+.mobile-menu-inactive {
+  visibility: hidden;
+  position: fixed;
+}
+
+.mobile-menu {
+  visibility: visible;
+  padding: 5%;
+  position: fixed;
+  left: 0;
+  top: 0;
+  overflow: hidden;
+  max-width: 767px;
+  min-width: 375px;
+  width: 100%;
+  height: 600px;
+  background-color: #6070ff;
+  opacity: 0.9;
+  z-index: 9;
+  transition: opacity 0.5s, visibility 0.5s;
+}
```

Comment:
- [ ] Ouch! this is so close but it seems like the `height: 600px;` you've added on the sidebar is preventing it from filling the entire screen as shown in the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39-122&t=cd5TWUpyWfE6T9vR-0)🎨. So please make sure to give it a `100vh` height 🎯.
- [ ] There seems to be a background blur happening on the background as shown in the design so please make sure to implement it:
    - Figma design:
      ![image](https://user-images.githubusercontent.com/104968236/231214474-802422ad-a906-49ae-a26e-545bedcc0919.png)
    - It could be a bit challenging but you can find out how it works after doing some research